healthz/config/cli.go

```diff
@@ -0,0 +1,73 @@
+package config
+
+import (
+	"time"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+	"github.com/gravitational/trace"
+)
+
+const (
+	EnvThresholds = "HEALTH_THRESHOLDS"
+	EnvAccessKey = "HEALTH_ACCESSKEY"
+	EnvListen = "HEALTH_LISTEN"
+	EnvInterval = "HEALTH_INTERVAL"
+	EnvLogLevel = "HEALTH_LOGLEVEL"
+	EnvCertFile = "HEALTH_CERTFILE"
+	EnvCAFile = "HEALTH_CAFILE"
+	EnvKeyFile = "HEALTH_KEYFILE"
+)
+
+const (
+	DocThresholds = "Specify thresholds for amount of dead nodes in cluster." +
+			"Format is `0=Y,Z=U`. It does mean that for cluster with size from " +
```

Comment:
instead of implementing custom formats for threshold, can you simply write a yaml file with structured format and parse it once? this will simplify configs. e.g. 

```yaml
checkers:
    nodes:
        alive:
        excluded: 
````